Sources/sourcekit-lsp/SourceKitLSP.swift

```diff
@@ -169,6 +175,12 @@ struct SourceKitLSP: ParsableCommand {
   )
   var indexPrefixMappings = [PathPrefixMapping]()
 
+  @Option(
+    name: .customLong("default-workspace-type"),
```

Comment:
Do we need to specify a `customLong` here? Shouldn’t it infer `default-workspace-type` based on the variable?